### PR TITLE
#162102645 landing page 

### PR DESCRIPTION
**Pull request by kirega**
**What does this PR do?**
It adds the ui layout and styles for the first page(landing page/index.html)

*Description of task to be completed?*
It creates the main navbar, a banner and footer.
It also has links to the login and sign up pages

*How should this be manually tested?*
Clone repo and open index.html in browser

*What are the relevant pivotal tracker stories?*
#162102645

*Relevant screenshots*

screenshot from 2018-11-21 23-57-32

### CODE REVIEW

File: ui/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        <header class="banner bg">
+            <div class="container">
+                <div class='banner-title'> Welcome to <span>iReporter</span>!</div>
+                <div class="banner-subtitle">Building a better nation starts with you</div>
+                <a href='login.html'>
+                    <button class="banner-button">WHISTLE BLOW</button>
+                </a>
+            </div>
+        </header>
+        <!-- <section class="landing-content flex-container ">
```
Comment (xcixor) [inline ui/index.html]: remove the commented code if no longer required

File: ui/styles.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    box-sizing: border-box;
+}
+
+body,
+html {
+    font-family: 'Lato', sans-serif;
+}
+.bg {
+    background: rgb(32, 52, 64);
+    background: linear-gradient(90deg, rgba(32, 52, 64, 1) 0%, rgba(184, 94, 78, 1) 50%, rgba(32, 52, 64, 1) 97%);
```
Comment (xcixor) [inline ui/styles.css:27]: great idea to use color gradient function to create a nice color transition

File: ui/index.html (hunk 1 above)
Comment (kirega) [reply ui/index.html]: I am on it.

File: ui/styles.css (hunk 2 above)
Comment (kirega) [reply ui/styles.css:27]: Thanks.
It was tough though getting the right mix of colors.
Luckily i got to use a cool free tool. https://cssgradient.io/ check it out sometime

**Comment by LarryKarani** [comment]
awesome font

**Pull request merged**